repos/spack_repo/builtin/packages/dbcsr/package.py

```diff
@@ -106,13 +106,7 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("+cuda", when="cuda_arch=none", msg=cuda_msg)
 
     dbcsr_amdgpu_targets = ("gfx906", "gfx910", "gfx90a", "gfx90a:xnack-", "gfx90a:xnack+")
-    amd_msg = f"DBCSR supports these AMD gpu targets:  {', '.join(dbcsr_amdgpu_targets)}"
-
-    conflicts(
-        "+rocm",
-        when="amdgpu_target=gfx942",
-        msg="DBCSR does not support gfx942. Please use amdgpu_target=gfx90a",
-    )
+    amd_msg = f"DBCSR supports these AMD gpu targets:  {', '.join(dbcsr_amdgpu_targets)}. Use amdgpu_target=gfx942 to enable gfx942 (mi300) support in DBCSR"
```

Comment:
```suggestion
    amd_msg = f"DBCSR supports these AMD GPU targets:  {', '.join(dbcsr_amdgpu_targets)}. Set 'amdgpu_target' explicitly to one of the supported targets."
```
? I think this would be a bit more general than just for Mi300.